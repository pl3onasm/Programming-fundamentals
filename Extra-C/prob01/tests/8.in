101 111 5896544442