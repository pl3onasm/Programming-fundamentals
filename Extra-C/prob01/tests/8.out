327192439030637667